modules/get-started/pages/release-notes/redpanda.adoc

```diff
@@ -62,7 +62,13 @@ The admin panel has been removed from the Redpanda Console UI. To manage users,
 
 == Iceberg improvements
 
-xref:manage:iceberg/about-iceberg-topics.adoc[Iceberg-enabled topics] now support custom partitioning for improved query performance, snapshot expiry, and a dead-letter queue for invalid records. Schema evolution is also supported with schema mutations implemented according to the Iceberg specification.
+xref:manage:iceberg/about-iceberg-topics.adoc[Iceberg-enabled topics] now support the following:
+
+- Custom partitioning for improved query performance.
```

Comment:
If any of these new updates have corresponding doc topics, please link to them here. thx

Reply:
Added xrefs